plugins/virtualenv/plugin.zsh

```diff
@@ -1,15 +1,34 @@
 # Color definitions
-GEOMETRY_COLOR_VIRTUALENV=${GEOMETRY_COLOR_PROMPT:-green}
+DEFAULT_COLOR=${GEOMETRY_COLOR_PROMPT:-green}
+GEOMETRY_COLOR_VIRTUALENV=${GEOMETRY_COLOR_VIRTUALENV:-$DEFAULT_COLOR}
+GEOMETRY_COLOR_CONDA=${GEOMETRY_COLOR_CONDA:-$DEFAULT_COLOR}
+GEOMETRY_VIRTUALENV_CONDA_SEPARATOR=${GEOMETRY_VIRTUALENV_CONDA_SEPARATOR:-:}
+
 
 geometry_prompt_virtualenv_setup() {}
 
 geometry_prompt_virtualenv_check() {
-  test $VIRTUAL_ENV || return 1
+    [ -n "${VIRTUAL_ENV}" -o -n "${CONDA_PREFIX}" ]
 }
 
 geometry_prompt_virtualenv_render() {
-  local ref=$(basename $VIRTUAL_ENV)
-  echo "$(prompt_geometry_colorize $GEOMETRY_COLOR_VIRTUALENV "(${ref})")"
+
+    if [ -n "${VIRTUAL_ENV}" ]; then
+
+        virtualenv_ref=$(basename $VIRTUAL_ENV)
+        virtualenv_str="$(prompt_geometry_colorize $GEOMETRY_COLOR_VIRTUALENV ${virtualenv_ref})"
+
+        if [ -n "${CONDA_PREFIX}" ]; then
```

Comment:
Something like this?
```bash
if [ -n "${VIRTUAL_ENV}" ]; then
    virtualenv_ref=$(basename $VIRTUAL_ENV)
    environment_str="$(prompt_geometry_colorize $GEOMETRY_COLOR_VIRTUALENV ${virtualenv_ref})"
fi

if [ -n "${VIRTUAL_ENV}" -a -n "${CONDA_PREFIX}" ]; then
    environment_str="${environment_str}${GEOMETRY_VIRTUALENV_CONDA_SEPARATOR}"
fi

if [ -n "${CONDA_PREFIX}" ]; then
    conda_ref="$(basename $CONDA_PREFIX)"
    environment_str="${environment_str}$(prompt_geometry_colorize $GEOMETRY_COLOR_CONDA ${conda_ref})"
fi
echo "${environment_str}"
```

Let me know if you'd like me to change it before merging.